src/layer/tile/GridLayer.js

```diff
@@ -413,15 +413,19 @@ L.GridLayer = L.Layer.extend({
 		if (!map) { return; }
 
 		if (center === undefined) { center = map.getCenter(); }
-		if (zoom === undefined) { zoom = Math.round(map.getZoom()); }
+		if (zoom === undefined) { zoom = map.getZoom(); }
 
 		if (zoom > this.options.maxZoom ||
 			zoom < this.options.minZoom) { return; }
```

Comment:
is there no side effect to changing the min/maxZoom logic so that you compare unrounded zoom?